.idea/workspace.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
do not commit .idea directory to the repo

Reply:
solved